Moodis/Moodis/Program.cs

```diff
@@ -11,15 +11,45 @@ namespace moodis
 {
     static class Program
     {
+        private const string WarningInRequest = "Connection failed. Is your internet turned on ?";
         /// <summary>
         /// The main entry point for the application.
         /// </summary>
         [STAThread]
+
         static void Main()
         {
-            Application.EnableVisualStyles();
-            Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new LoginForm());
+
+            if(checkForInternet())
+            {
+                Application.EnableVisualStyles();
+                Application.SetCompatibleTextRenderingDefault(false);
+                Application.Run(new LoginForm());
+            }
+            else
+            {
+                //Application.Exit();
+                MessageBox.Show(WarningInRequest);
+                return;
+            }
+            
+            
+        }
+
+        private static bool checkForInternet()
```

Comment:
This should work without `static`

Reply:
Plot twist:


















It does not 😮

Reply:
Oh right, Main is also static. my bad

Reply:
This class starts the program and must be static, so we cannot change it